### PR TITLE
Project - MeltShiny

### PR DESCRIPTION
**Pull request by LikhitaPulijala**
Added info related to MeltShiny project.

### CODE REVIEW

File: docs/MeltShiny/about.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+id: about
+title: MeltShiny(Meltwin)
+custom_edit_url: https://github.com/oss-slu/MeltWin2.0
+---
+
+## Overview
+
+MeltShiny offers an intuitive graphical interface, eliminating the need for the console and accommodating users of all skill levels. Leveraging MeltR's robust calculations, it provides automated processing of input files and outlier removal from Van't Hoff plots and results tables. This optimizes researchers' workflows, minimizing computer interaction time and maximizing dedicated time for research and analysis.
+
```
Comment (kate-holdener) [inline docs/MeltShiny/about.md:10]: @LikhitaPulijala We need a more high-level overview of the project, what it's going to be used for, and its value. The idea is that someone who is reading about this project for the first time would get excited about it.

**Review by kungfuchicken — DISMISSED** [review]
Good enough for now.

Needs additional work after the big branch is merged.

**Comment by kungfuchicken** [comment]
This should be against the `projects-fall2023` branch, not `main`.

**Comment by kungfuchicken** [comment]
closing and re-opening.

**Pull request closed without merging**